engine/pkg/client/internal/call.go

```diff
@@ -92,6 +94,7 @@ func (c *Call[ReqT, RespT, F]) callOnce(ctx context.Context) (RespT, error) {
 }
 
 func (c *Call[ReqT, RespT, F]) isRetryable(errIn error) bool {
+	log.Error("call rpc returns error", zap.Any("request", c.request), zap.Error(errIn))
```

Comment:
Does it cause duplicate error logs? It seems we have printed error logs in most scenarios.

Reply:
No, `errrIn` is ignored by the retry package. However, if multiple retries fail, the same error may be printed here.

Reply:
Can we only log retriable errors here?

Reply:
If an error is not retriable, it will be printed only once.

Reply:
Not exactly. Most outside may print the log as well. See https://github.com/pingcap/tiflow/blob/19365b0790e614d318fa95f7ab6705273913dd66/engine/executor/server.go#L750.